teams/lang-docs.toml

```diff
@@ -0,0 +1,22 @@
+name = "lang-docs"
+subteam-of = "lang"
+
+[people]
+leads = ["ehuss", "JohnTitor"]
+members = [
+    "ehuss",
+    "Havvy",
+    "JohnTitor",
```

Comment:
@ehuss @havvy @matthewjasper Wanted to make sure y'all were okay with this; I do not see many PRs from @JohnTitor on the nomicon and zero reviews.

It's atypical to add new maintainers in such scenarios, typically team members get added once they've had some involvement already.

Of course, if @JohnTitor has been involved in other, less visible ways, and y'all think they'd be a good addition to the team, ignore me.